CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ file(GLOB headers RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}"
     "pywinrt/${CMAKE_PROJECT_NAME}/src/*.h"
 )
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
While we are doing this, let's remove the `/await` from the `add_compile_options()` below (it is for coroutines in c++17 and isn't needed anymore in c++20).